docset/winserver2012r2-ps/addsadministration/Remove-ADClaimType.md

```diff
@@ -43,7 +43,7 @@ This command removes the claim type with the name Title.
 
 ### Example 2: Get all disabled claim types and remove them
 ```
-PS C:\>Get-ADClaimType -Filter { Enabled -eq $FALSE } | Remove-ADClaimType
+PS C:\>Get-ADClaimType -Filter "Enabled -eq $False" | Remove-ADClaimType
```

Comment:
Based on other command lines in this PR, I wonder if **$False** should be enclosed in backticks here.